index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/PublicationDetails.java

```diff
@@ -29,6 +31,7 @@ public static final class Builder {
         private String identifier;
         private String type;
         private String title;
+        private String abstractText;
```

Comment:
Maybe simply abstract?

Reply:
`abstract` is a reserved keyword in Java, and can't be used as variable name

Reply:
That's why i set the JsonProperty as "abstract": `@JsonProperty("abstract")`